os/arch/arm/include/armv7-a/syscall.h

```diff
@@ -85,23 +85,23 @@
 
 #ifndef CONFIG_SYS_RESERVED
 #error "CONFIG_SYS_RESERVED must be defined to have the value"
-#endif //CONFIG_SYS_RESERVED
+#endif							//CONFIG_SYS_RESERVED
```

Comment:
some places it is used with /**/ and in some places // is used

Reply:
I think this is ok. It was like this in the original code. We have only changed the indentation.